core/cmd/app.go

```diff
@@ -717,6 +723,17 @@ func NewApp(client *Client) *cli.App {
 							Usage:  "List all chains",
 							Action: client.IndexChains,
 						},
+						{
+							Name:   "configure",
+							Usage:  "Configure an EVM chain",
+							Action: client.ConfigureChain,
+							Flags: []cli.Flag{
+								cli.Int64Flag{
+									Name:  "id",
```

Comment:
this could be `chain-id` to be consistent with other commands

Reply:
It's `-chain-id` under `nodes`, I think `chains evm -id` is clear enough (compared to `chains evm -chain-id` or `chains evm -evm-chain-id` which is more to type)